src/debugger/launcher.ts

```diff
@@ -49,14 +49,17 @@ export class Launcher {
                         // We've seen that if we don't prewarm the bundle cache, the app fails on the first attempt to connect to the debugger logic
                         // and the user needs to Reload JS manually. We prewarm it to prevent that issue
                         .then(() => {
+                            Log.logMessage("Prewarming bundle cache. This may take a while...");
                             generator.step("prewarmBundleCache");
                             return extensionMessageSender.sendMessage(em.ExtensionMessage.PREWARM_BUNDLE_CACHE, [runOptions.platform]);
                         })
                         .then(() => {
+                            Log.logMessage("Building and running application.");
                             generator.step("mobilePlatform.runApp");
                             return mobilePlatform.runApp(runOptions);
                         })
                         .then(() => {
+                            Log.logMessage("Starting debugging session.");
```

Comment:
From the user perspective the Debugging Session started when he pressed the button. I'd change this to something along the lines of: "Starting the worker which will run the react-native application logic."